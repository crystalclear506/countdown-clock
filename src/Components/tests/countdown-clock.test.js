import { expect, test } from '@jest/globals';
import React from 'react';
import renderer from 'react-test-renderer';
import CountdownClock from '../countdown-clock';


test('Component should rendered correctly', () => {
    const countdownClock = renderer.create(
        <CountdownClock/>
    );
    expect(countdownClock).toBeDefined();
});