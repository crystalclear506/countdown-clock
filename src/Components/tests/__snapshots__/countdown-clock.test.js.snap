// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should be rendered correctly 1`] = `
<div
  className="clock-container"
>
  <div
    className="clock"
  >
    <div
      className="cell"
    >
      00
    </div>
    <div
      className="colons"
    >
      <div>
        :
      </div>
    </div>
    <div
      className="cell"
    >
      00
    </div>
    <div
      className="colons"
    >
      <div>
        :
      </div>
    </div>
    <div
      className="cell"
    >
      00
    </div>
    <div
      className="colons"
    >
      <div>
        :
      </div>
    </div>
    <div
      className="cell"
    >
      000
    </div>
  </div>
</div>
`;
